### ISSUE TITLE
Documentation for scipy.stats.norm.fit is incorrect

### ISSUE THREAD

**Issue by ngoldbaum** [scipy/scipy#6727]
Currently, this function says it should return a 3-tuple:

```
In [1]: import scipy.stats as s

In [2]: print(s.norm.fit.__doc__)
*snip*
        Returns
        -------
        shape, loc, scale : tuple of floats
            MLEs for any shape statistics, followed by those for location and
            scale.
*snip*

In [5]: scipy.__version__
Out[5]: '0.18.1'
```

But the normal distribution is a special case, and its `fit` implementation [only returns a 2-tuple](https://github.com/scipy/scipy/blob/master/scipy/stats/_continuous_distns.py#L185).

**Comment by ev-br** [comment]
Sigh.

To explain (definitely not to defend) what's going on: these method docstrings are inherited verbatim from `rv_continuous`. Since stats distributions are instances, we rewrite docstrings on import, when instances are constructed. This way we can customize docstrings of the instances themselves (this is what ends up in e.g. https://docs.scipy.org/doc/scipy/reference/generated/scipy.stats.norm.html). Docstrings of instance methods, however, are not writeable under python 2 at least. So they end up being generic, including mysteries like (check `norm.pdf.__doc__`)

```
arg1, arg2, arg3,... : array_like
      The shape parameter(s) for the distribution (see docstring of the instance object for more information).
```

Basically, the best one can do is to *add* some clarifying info to the docstring of `rv_continuous.fit`. A pull request would be gladly accepted.
